lms/static/sass/views/_login-register.scss

```diff
@@ -111,51 +97,35 @@ $sm-btn-linkedin: #0077b5;
 
             .text {
                 position: relative;
-                background: white;
-                padding: 0 10px;
-                z-index: 6;
+                top: -2px; // Aligns center of text with center of line (CR)
+                padding: 0 $baseline;
+                background: $white;
                 text-transform: none;
-                font-size: 0.7em;
-                font-weight: 600;
+                z-index: 6;
             }
         }
     }
 
     .nav-btn {
-        margin: 0 28px;
-        padding: 7px 0;
-        width: calc( 100% - 56px );
-        border: 3px solid $m-blue-d5;
-        border-radius: 5px;
-        color: $m-blue-d5;
-        box-shadow: none;
-        text-shadow: none;
+        @extend %btn-secondary-blue-outline;
+        width: 100%;
+        height: ($baseline*2);
         text-transform: none;
-        background: white;
-
-        &:hover {
-            background: white;
-            border-color: $m-blue-d6;
-            color: $m-blue-d6;
-        }
-
-        &:active {
-            box-shadow: none;
-        }
+        text-shadow: none;
+        font-weight: 600;
+        letter-spacing: normal;
     }
 
     .form-type {
         @include box-sizing(border-box);
-        width: 330px;
+        width: flex-grid(6);
```

Comment:
Yeah... Once I saw it in place with the updated styles, I changed my mind because it seemed better at the only slightly bigger size. But also using percentages for the width without a min-width makes it scrunch down too far.